src/Evolution/DiscontinuousGalerkin/FilterTags.hpp

```diff
@@ -0,0 +1,49 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "Options/Options.hpp"
+
+namespace OptionTags {
```

Comment:
Same question about this being evolution-specific. Probably best to make a directory in `NumericalAlgorithms` where you can put this `Tags.hpp`